R/fontset-dejavu.R

```diff
@@ -0,0 +1,54 @@
+#' @references \url{http://dejavu-fonts.org}
+#' @include fontset.R
+fontset_dejavu <- function(variant = "sans", style = "book") {
+  if (system.file("fonts", package = "fontHeavy") == "") {
```

Comment:
At some point it'll get too heavy even for github? One font per package seems like a simpler and more generalisable system.